modules/virt-about-preallocation.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-about-preallocation_{context}"]
+= About preallocation
+
+The Containerized Data Importer (CDI) can use the QEMU preallocate mode for data volumes in order to improve write performance. Preallocation mode can be used for importing and uploading operations and when creating blank data volumes.
+
+If preallocation is enabled, the CDI uses the better preallocation method depending on the underlying file system and device type.
+
+`fallocate`::
+If the file system supports it, CDI will use the `fallocate` call of the operating system to preallocate space using the `posix_fallocate` function, which allocates blocks and marks them as uninitialized. 
```

Comment:
s/will use/uses

The new version of the IBM style guide seems cool with possessives (pg. 71 on the source), so maybe "the operating system's `fallocate` call" would be better here.

("The call of the operating system" reminds me of "the call of the wild." :))

s/using/by using

Reply:
Wow, really? Gosh, that might take some getting used to.
The call of the OS, it sings to me..

Reply:
I know, it's so bizarre! I'm wondering what other things of significance have changed in the new version...